website/docs/core-library/compilation-options.md

```diff
@@ -56,7 +56,7 @@ Should be set to `Release` for production builds, and `Debug` in development.
 
 ## ENABLE_COVERAGE
 
-Whether to compile `Debug` builds with coverage instrumentation (compatible with GCC, Clang, and lcov).
+Whether to compile `Debug` builds with coverage instrumentation (compatible with GCC, Clang, and lcov). This also elides defensive code in the library.
```

Comment:
I had to look up "elides". Maybe "bypasses"? 😅 

And maybe even a quick note on why:

"This also bypasses defensive code in the library, since it should typically not be reachable by coverage."